### PR TITLE
Made it so when user logged in, Header is replaced with LoggedInHeader.

### PR DESCRIPTION
**Pull request by ericmiller777**
Made it so when user logged in, Header is replaced with LoggedInHeader

screenshotofloggedinheader

### CODE REVIEW

File: src/AddProjectsPage/AddProjectsPage.tsx [hunk 1]
```diff
@@ -71,9 +74,13 @@ class AddProjectsPage extends React.Component<PassedProps, State> {
...
 
   render() {
+    let isUserLoggedIn = false;
+    if (this.props.user.email) {
+      isUserLoggedIn = true;
+    }
     return (
       <div>
-        <Header />
+        {isUserLoggedIn === true ? <LoggedInHeader /> : <Header />}
```
Comment (serpient) [inline src/AddProjectsPage/AddProjectsPage.tsx]: Line 83, this code is repeated in all the files that require a header. Is there a way to make it so the conditional is used once in the project? And the correct header is passed to all the pages requiring a header?

**Comment by serpient** [comment]
image

stylistic changes:
- Choose a portal should be centered in the overall header. 
- The grey circle behind the user icon is still showing up. this should be removed

**Comment by serpient** [comment]
image

- Choose a portal should be centered in the overall header. 
- The grey circle behind the user icon is still showing up. this should be removed

File: src/Projects Page/ProjectsPage.tsx [hunk 2]
```diff
@@ -1,49 +1,60 @@
...
+  constructor(props: PassedProps) {
+    super(props);
+    this.state = {
+      searchTerm: ''
+    };
+  }
+
+  public onFormChange(e: React.FormEvent<HTMLButtonElement>): void {
+    this.setState({
+      [e.currentTarget.name]: e.currentTarget.value
```
Comment (serpient) [inline src/Projects Page/ProjectsPage.tsx]: we found out this method of setting name and value doesn't work.
use this as an example (also found in login/register.tsx)

```
handleChange = (e: React.FormEvent<HTMLInputElement>): void => {
    var { name, value } = e.currentTarget;

    this.setState({
      [name]: value
      // tslint:disable-next-line
    } as any);
  };
```

**Comment by serpient** [comment]
image

- cloud is showing up above the pop-up. z-index needs to be adjusted
- horizontal line styling needs to be a light grey

File: src/AddProjectsPage/AddProjectsPage.tsx [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 import Header from '../Header';
 import Footer from '../Footer';
 import '../styles/AddProjectsPage.css';
-import { PassedProps, State, Props } from '../types/AddProjectsPage.d';
+import { Store } from '../types/Redux';
+import { PassedProps, State } from '../types/AddProjectsPage.d';
 
 class AddProjectsPage extends React.Component<PassedProps, State> {
-  constructor(props: Props) {
+  constructor(props: PassedProps) {
```
Comment (luoto) [inline src/AddProjectsPage/AddProjectsPage.tsx]: I think we could go for more descriptive name for this one.

**Comment by serpient** [comment]
image


instead of shrinking the test like that, perhaps we should try and bundle everything but the logo in a menu icon, that drops down to show everything
like this:
image

**Comment by serpient** [comment]
another example
image

**Comment by serpient** [comment]
ur eric-miller-test seems to have a few compile errors, so it never logs in. do you see these errors?
image
image

**Comment by serpient** [comment]
approving for now as Login cannot be tested currently

**Pull request merged**